### PR TITLE
Upgrade Tensorflow to 2.2.0

### PR DESCRIPTION
**Pull request by maresb**

### CODE REVIEW

**Comment by romainx** [comment]
Hello @maresb,

Thank you, nothing to add, all good 😁.

Thank you.

**Comment by romainx** [comment]
Hello @maresb ,

As suggested in the PR #1082 we should use the `pip --no-cache-dir` option to reduce the image size. Maybe you could add the option in your PR to perform both changes in a row?

Many thanks.

File: tensorflow-notebook/Dockerfile [hunk 1]
```diff
@@ -7,6 +7,6 @@ LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 
 # Install Tensorflow
 RUN pip install --quiet \
```
Comment (romainx) [inline tensorflow-notebook/Dockerfile:9]: Could you please add the `--no-cache-dir` option in the `pip install` command.

```diff
- RUN pip install --quiet \
+ RUN pip install --quiet --no-cache-dir \
```
Many thanks

**Comment by maresb** [comment]
Thank you @mathbunnyru! Integrated with #1082.

**Comment by maresb** [comment]
I'd actually feel better about this if we did it the other way around. @mathbunnyru's patch is so awesome, so have him bump the version so that he gets proper credit for #1082.

**Comment by mathbunnyru** [comment]
Thank you so much, @maresb !

I increased the version in my PR as you suggested :)

**Pull request closed without merging**